core/src/main/java/com/sequenceiq/cloudbreak/converter/v4/stacks/cli/NetworkToNetworkV4RequestConverter.java

```diff
@@ -19,27 +19,24 @@
 @JsonInclude(Include.NON_NULL)
 public class NetworkToNetworkV4RequestConverter {
 
+    private static final Logger LOGGER = LoggerFactory.getLogger(NetworkToNetworkV4RequestConverter.class);
+
     @Inject
     private ProviderParameterCalculator providerParameterCalculator;
 
     public NetworkV4Request convert(Network source) {
+        throwIfNull(source, () -> new IllegalStateException(Network.class.getSimpleName() + " should not be null!"));
         NetworkV4Request networkRequest = new NetworkV4Request();
         if (source.getAttributes() != null) {
             providerParameterCalculator.parse(cleanMap(source.getAttributes().getMap()), networkRequest);
         }
         networkRequest.setSubnetCIDR(source.getSubnetCIDR());
+        if (networkRequest.isEmpty()) {
+            LOGGER.debug("The created {} is empty, therefore null shall return.", NetworkV4Request.class.getSimpleName());
+            return null;
```

Comment:
Could you please create tests for the return value?

Reply:
have test for this: `testConvertWhenEmptyNetworkInstancePassedThenNullShouldReturn`